src/modules/ce/components/project/ProjectCeReferralsPage.tsx

```diff
@@ -1,48 +1,76 @@
 import React, { useMemo } from 'react';
+import ButtonLink from '@/components/elements/ButtonLink';
 import CommonTabs from '@/components/elements/CommonTabs';
 import PageTitle from '@/components/layout/PageTitle';
 import ProjectOpportunitiesTable from '@/modules/ce/components/project/ProjectOpportunitiesTable';
 import ProjectReferralsTable from '@/modules/ce/components/project/ProjectReferralsTable';
 import { useProjectDashboardContext } from '@/modules/projects/components/ProjectDashboard';
+import { ProjectDashboardRoutes } from '@/routes/routes';
+import { generateSafePath } from '@/utils/pathEncoding';
 
 const ProjectCeReferralsPage: React.FC = () => {
   const { project } = useProjectDashboardContext();
 
+  const projectSendsDirectReferrals =
+    project.coordinatedEntryFeatures?.sendsDirectReferrals;
+  const userCanSendReferrals = project.access.canManageOutgoingReferrals;
+
   const tabDefinitions = useMemo(() => {
     const defs = [];
 
-    // TODO(#7321): only render referrals tab on this page if this project has CE referral feature enabled (either directly or via waitlist)
-    if (project.access.canViewReferrals || project.access.canViewOwnReferrals) {
+    const projectSupportsReferrals =
+      project.coordinatedEntryFeatures?.supportsReferrals;
+    const userCanViewReferrals =
+      project.access.canViewReferrals || project.access.canViewOwnReferrals;
+    if (projectSupportsReferrals && userCanViewReferrals) {
       defs.push({
         title: 'Referrals',
         key: 'referrals',
         contents: <ProjectReferralsTable projectId={project.id} />,
       });
     }
 
-    // TODO(#7321): only render units tab on this page if this project uses waitlist-based referral creation. It doesn't make sense to link to unit from here if it doesn't have waitlist.
-    if (project.access.canViewUnits) {
+    // Only render units tab on this page if this project uses waitlist-based referral creation.
+    // It doesn't make sense to link to unit from here if it doesn't have waitlist.
+    const projectSupportsWaitlistReferrals =
+      project.coordinatedEntryFeatures?.supportsWaitlistReferrals;
+    const userCanViewUnits = project.access.canViewUnits;
+    if (projectSupportsWaitlistReferrals && userCanViewUnits) {
       defs.push({
         title: 'Available Units',
         key: 'available-units',
         contents: <ProjectOpportunitiesTable projectId={project.id} />,
       });
     }
 
-    // TODO(#7321) if project can send direct referrals (or has any sent referrals?), display table of outgoing referrals
-    // {
-    //   title: 'Sent Referrals',
-    //   key: 'sent-referrals',
-    //   contents: null, // Placeholder for future implementation
-    // }
-
+    if (projectSendsDirectReferrals && userCanSendReferrals) {
+      defs.push({
+        title: 'Sent Referrals',
+        key: 'sent-referrals',
+        contents: 'Placeholder', // Placeholder for future implementation
```

Comment:
To discuss: Do we consider this safe to ship, because it won't be shown unless a project has a Sends Direct Referrals config, which won't happen in non-CE installations? 
<img width="2442" height="726" alt="Screenshot 2025-07-16 at 11 12 45 AM" src="https://github.com/user-attachments/assets/e1e6a21e-e05a-49b1-8f6e-9a2b7e6f798c" />

Reply:
I'd prefer to leave this commented-out since an administrator could enable it